utils/supported-browsers-from-mdn.mjs

```diff
@@ -13,16 +13,15 @@ const MDNToBrowserlist = {
 
 function getBrowsersFromFeature(mdnConfigPath, feature) {
 	const mdnFeature = _get(mdn, mdnConfigPath);
+	if (!mdnFeature) {
+		throw new Error(`Invalid mdn config path "${mdnConfigPath}" in feature "${feature}"`);
```

Comment:
Easier to debug if incorrect paths throw.
MDN is changing these a lot more than I would expect.